### PR TITLE
Further fixes to the rig system, now with a new branch and right repository.

### PR DESCRIPTION
**Pull request by MarsM0nd**
Added fixes:
Stops wearable parts from getting removed by hand.
Stripping the unsealed rig while the wearer is incapacited correctly retracts the parts.
The surgery ends and has a step description on the surgical computer.

Interesting how that the github desktop did make the PR on the own repository.

### CODE REVIEW

**Comment by Spacemanspark** [comment]
Intentation isn't a word.

File: code/modules/surgery/rig_removal.dm [hunk 1]
```diff
@@ -1,25 +1,26 @@
...
-
-/datum/surgery/rigsuit/can_start(mob/user, mob/living/carbon/target)
-  if(ishuman(target))
-    var/mob/living/carbon/human/H = target
-    var/obj/item/backitem = H.get_item_by_slot(slot_back)
-    if(istype(backitem,/obj/item/weapon/rig)) //Check if we have a rig to operate on
-      if(backitem.flags&NODROP) //Check if the rig is sealed, if not, we don't need to operate
-        return 1
-  return 0
+datum/surgery/rigsuit
```
Comment (FlattestGuitar) [inline code/modules/surgery/rig_removal.dm]: :eyes:
`/`

File: code/modules/surgery/rig_removal.dm [hunk 2]
```diff
@@ -1,25 +1,26 @@
...
+	name = "Rig Unsealing"
+	steps = list(/datum/surgery_step/rigsuit)
+	possible_locs = list("chest")
+
+datum/surgery/rigsuit/can_start(mob/user, mob/living/carbon/target)
```
Comment (FlattestGuitar) [inline code/modules/surgery/rig_removal.dm]: `/`

File: code/modules/surgery/rig_removal.dm [hunk 3]
```diff
@@ -1,25 +1,26 @@
...
+	if(istype(target,/mob/living/carbon/human))
```
Comment (FlattestGuitar) [inline code/modules/surgery/rig_removal.dm]: `if(ishuman(target))`

**Comment by FlattestGuitar** [comment]
I'm getting a déjà vu here...

Fix those and I'll merge it.

**Comment by MarsM0nd** [comment]
Not sure why/how i did revert so much in the rig surgery, my usage of git summarised: [xkcd git](http://imgs.xkcd.com/comics/git.png)

**Comment by FlattestGuitar** [comment]
👀

**Pull request merged**